openstudiocore/resources/analysis/pat_export/scripts/StandardReports/measure.rb

```diff
@@ -1,139 +1,165 @@
 require 'erb'
+require 'json'
+
+require "#{File.dirname(__FILE__)}/resources/os_lib_reporting"
+require "#{File.dirname(__FILE__)}/resources/os_lib_schedules"
+require "#{File.dirname(__FILE__)}/resources/os_lib_helper_methods"
 
 #start the measure
-class StandardReports < OpenStudio::Ruleset::ReportingUserScript
+class OpenStudioResults < OpenStudio::Ruleset::ReportingUserScript
```

Comment:
I was expecting to see a new measure (OpenStudio Results) listed here:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/bcl/BCLMeasure.hpp#L101

Then, code to update existing PAT projects here:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/analysisdriver/SimpleProject.cpp#L442

This can get pretty tricky, can we just have the measure on BCL for the moment and not include in the package for 1.8.1?  The files in the measure should be renamed to match the class name in the measures repo (if that is not already the case).